control

```diff
@@ -0,0 +1,7 @@
+Package: bob
+Version: current_version
+Section: unknown
+Priority: optional
+Architecture: amd64
+Maintainer: qoollo.com <info@qoollo.com>
```

Comment:
Set maintainer to `Qoollo` (without `.com`).

Additionally, you can add `Homepage` section that references to Bob repository on GitHub